packages/framework/src/Model/Article/ArticleRepository.php

```diff
@@ -149,4 +219,20 @@ public function getAllByDomainId($domainId)
             'domainId' => $domainId,
         ]);
     }
+
+    /**
+     * @param int $domainId
+     * @param string $placement
+     *
```

Comment:
```suggestion
```

Reply:
Why do you want to remove this? I stopped using it after previous review and this one is forgotten.

Anyway IMO it is more pretty without this empty line.

We definitely should enable rule for sniffing/fixing it (no matter what style of this phpdoc wi will want).

Reply:
fixed